README.md

```diff
@@ -36,7 +36,7 @@ Once the `node` and `npm` are installed, run this command to install all the req
 at the correct versions:
 
 ```bash
-npm install
+npm install; npm install natives
```

Comment:
There is a huge warning about this package: https://www.npmjs.com/package/natives

Better to go with the recommandation and install `graceful-fs` instead